content/en/docs/setup/production-environment/tools/kubeadm/troubleshooting-kubeadm.md

```diff
@@ -407,4 +407,12 @@ be advised that this is modifying a design principle of the Linux distribution.
 
 This error message is shown when upgrading a Kubernetes cluster with `kubeadm` in the case of running an external etcd. This is not a critical bug and happens because older versions of kubeadm perform a version check on the external etcd cluster. You can proceed with `kubeadm upgrade apply ...`. 
 
-This issue is fixed as of version 1.19. 
\ No newline at end of file
+This issue is fixed as of version 1.19.
+
+## `kubeadm reset` unmounts `/var/lib/kubelet`
+
+If `/var/lib/kubelet` is being mounted, performing a `kubeadm reset` will effectively unmount it.
```

Comment:
```suggestion
If `/var/lib/kubelet` is a mount point, `kubeadm reset` unmounts that filesystem as a side effect.
```
?

What does “effectively” mean here? Is something other than `unmount()` happening?

Reply:
only unmount is happening